File: files/fileSystemUsing.cpp
#include<iostream>
#include<fstream>
using namespace std;
int main(){
	ofstream file("C:\\Users\\Seyrek\\Desktop\\C++\\files\\text.txt");
	string str;
	cout << "Enter a text" << endl;
	cin >> str;
	file << str << endl;
	file.close();
	return 0;
}

File: files/fileSystemUsing.java
package springIntro;

import java.io.*;
import java.util.Scanner;

public class fileSystemUsing {

	public static void main(String[] args) throws IOException {
		File file = new File("C:\\Users\\Seyrek\\eclipse-workspace\\springIntro\\src\\springIntro\\text.txt");
		file.createNewFile();//Dosyayi olustur
		FileWriter print = new FileWriter(file);
		Scanner in = new Scanner(System.in);
		System.out.println("Text to write to file");
		print.write(in.next());
		print.flush();//Veri akisini temizle
		print.close();//Dosyayi kapat
	}

}
